src/main/java/org/opentripplanner/standalone/config/routerequest/MappingConfig.java

```diff
@@ -0,0 +1,39 @@
+package org.opentripplanner.standalone.config.routerequest;
+
+import static org.opentripplanner.standalone.config.framework.json.EnumMapper.docEnumValueList;
+import static org.opentripplanner.standalone.config.framework.json.OtpVersion.V2_0;
+import static org.opentripplanner.standalone.config.framework.json.OtpVersion.V2_6;
+
+import org.opentripplanner.routing.api.request.preference.MappingFeature;
+import org.opentripplanner.routing.api.request.preference.MappingPreferences;
+import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
+
+public class MappingConfig {
+
+  public static void mapMappingParams(
+    String parameterName,
+    NodeAdapter root,
+    MappingPreferences.Builder builder
+  ) {
+    NodeAdapter c = root
+      .of(parameterName)
+      .since(V2_6)
+      .summary("Configure mapping of the internal data structures into itineraries.")
+      .asObject();
+
+    if (c.isEmpty()) {
+      return;
+    }
+
+    builder
+      .withOptInFeatures(
+        c
+          .of("optInFeatures")
+          .since(V2_0)
```

Comment:
2.6?

Reply:
Fixed in e17f2cbc64909d98a216a29bbece71281ad2204f